db/views/national_hearing_queue_entries_v08.sql

```diff
@@ -0,0 +1,171 @@
+  REPLACE (CAST(appeals.receipt_date AS TEXT), '-', '') AS receipt_date,
+  CAST(appeals.uuid AS TEXT) AS external_id,
+  CAST(appeals.stream_type AS TEXT) AS appeal_stream,
+  CAST(appeals.stream_docket_number AS TEXT) AS docket_number,
+  CASE
+    WHEN appeals.aod_based_on_age = TRUE
+      OR advance_on_docket_motions.granted = TRUE
+      OR veteran_person.date_of_birth <= CURRENT_DATE - INTERVAL '75 years'
+      OR aod_based_on_age_recognized_claimants.quantity > 0
+    THEN TRUE
+    ELSE FALSE
+  END AS aod_indicator,
+  tasks.id AS task_id,
+  tasks.assigned_to_id AS assigned_to_id,
+  tasks.assigned_to_type AS assigned_to_type,
+  tasks.assigned_at AS assigned_at,
+  tasks.assigned_by_id AS assigned_by_id,
+  CASE
+  	WHEN tasks.status = 'on_hold' THEN CURRENT_DATE - tasks.placed_on_hold_at::date
+  	ELSE null
+  END AS days_on_hold,
+  COALESCE(tasks.closed_at::date, CURRENT_DATE) - tasks.assigned_at::date AS days_waiting,
+  tasks.status AS task_status,
+  CASE
+    WHEN stream_type = 'court_remand'
+     OR (
+	     CASE
+	    	WHEN appeals.aod_based_on_age = TRUE
+		      OR advance_on_docket_motions.granted = TRUE
+		      OR veteran_person.date_of_birth <= CURRENT_DATE - INTERVAL '75 years'
+		      OR aod_based_on_age_recognized_claimants.quantity > 0
+	    	THEN TRUE
+	    	ELSE FALSE
+  		END
+     ) IS TRUE
+     OR receipt_date <= COALESCE((SELECT cutoff_date FROM latest_cutoff_date),'2019-12-31')
+    THEN TRUE
+    ELSE FALSE
+  END AS schedulable,
+  veterans.state_of_residence,
+  veterans.country_of_residence,
+  cached_appeal_attributes.suggested_hearing_location,
+  CASE
+    WHEN (request_issues_status.aggregate_mst_status = true) THEN true
+    ELSE false
+  END AS mst_indicator,
+  CASE
+    WHEN (request_issues_status.aggregate_pact_status = true) THEN true
+    ELSE false
+  END AS pact_indicator
+FROM
+  appeals
+  JOIN tasks ON tasks.appeal_type = 'Appeal'
+  AND tasks.appeal_id = appeals.id
+  LEFT JOIN advance_on_docket_motions ON advance_on_docket_motions.appeal_id = appeals.id
+  JOIN veterans ON appeals.veteran_file_number = veterans.file_number
+  LEFT JOIN people veteran_person ON veteran_person.participant_id = veterans.participant_id
+  LEFT JOIN LATERAL (
+    SELECT count(*) as quantity
+    FROM claimants
+    JOIN people ON claimants.participant_id = people.participant_id
+    WHERE claimants.decision_review_id = appeals.id
+      AND claimants.decision_review_type = 'Appeal'
+      AND people.date_of_birth <= CURRENT_DATE - INTERVAL '75 years'
+  ) aod_based_on_age_recognized_claimants ON TRUE
+  LEFT JOIN cached_appeal_attributes ON (
+    cached_appeal_attributes.appeal_id = appeals.id AND cached_appeal_attributes.appeal_type = 'Appeal'
+  )
+  LEFT JOIN (
+    SELECT decision_review_id, bool_or(mst_status) AS aggregate_mst_status, bool_or(pact_status) AS aggregate_pact_status
+    FROM request_issues
+    WHERE decision_review_type = 'Appeal'
+    GROUP BY decision_review_id
```

Comment:
```suggestion
    SELECT decision_review_id, bool_or(mst_status) AS aggregate_mst_status, bool_or(pact_status) AS aggregate_pact_status
    FROM request_issues
    WHERE decision_review_type = 'Appeal'
      AND (mst_status IS TRUE OR pact_status IS TRUE)
    GROUP BY decision_review_id
```
According to `EXPLAIN (ANALYZE, TIMING)`, whenever adding in the extra constraint it halves the execution time (this was done against the Prod Replica DB in Metabase).